utils/scripts/ocm/ocm.py

```diff
@@ -1186,6 +1237,55 @@ def compare_with_old_version_file(self):
     subparsers = parser.add_subparsers(
         title="Available sub commands", help="Available sub commands"
     )
+    # Argument of update_ocm_channel
+    update_ocm_policy = subparsers.add_parser(
+        "update_ocm_policy",
+        help="Parser to get osd chnages in json file",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+    update_ocm_policy.add_argument(
+        "--update-policies-json",
+        help="pass the json file to update ocm policy",
+        action="store",
+        dest="update_policies_json",
+        required=True,
+    )
+    update_ocm_policy.add_argument(
+        "--cluster-name",
+        help="osd cluster name",
+        action="store",
+        dest="cluster_name",
+        metavar="",
+        default="osd-qe-1",
+    )
+    update_ocm_policy.set_defaults(
+        func=ocm_obj.update_ocm_policy
+    )
+
+    # Argument of update_ocm_channel
+    update_ocm_channel = subparsers.add_parser(
+        "update_ocm_channel",
+        help="Parser to get osd chnages in json file",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+    update_ocm_channel.add_argument(
+        "--update-ocm-channel-json",
+        help="pass json file to update ocm channel",
+        action="store",
+        dest="upadte_ocm_channel_json",
+        required=True,
+    )
+    update_ocm_channel.add_argument(
+        "--cluster-name",
+        help="osd cluster name",
+        action="store",
+        dest="cluster_name",
+        metavar="",
+        default="osd-qe-1",
```

Comment:
Use the default name of cluster that is present in create_cluster.

Reply:
replaced with qeaisrhods-xyz